R/check-class.R

```diff
@@ -102,17 +108,44 @@ check_update <- function(self, private) {
 
 check_web <- function(self, private, which) {
 
+  ids <- select_ids(which = which, self = self, 
+                    private = private)
+
+  urls <- paste0(sub("/api$", "/status/", baseurl()), ids)
+  
+  lapply(urls, browseURL)
+  invisible(self)
+}
+
+check_urls <- function(self, private, which) {
+  
+  ids <- select_ids(which = which, self = self, 
+                    private = private)
+  
+  urls <- paste0(sub("/api$", "/status/", baseurl()), ids)
+  
+  do.call("rbind",
+          lapply(ids, get_check_urls))
+}
+
+get_check_urls <- function(id){
+  data.frame(html = paste0(sub("/api$", "/status/", baseurl()), id),
+             text = paste0(sub("/api$", "/status/original/", baseurl()), id),
+             artifacts = paste0("https://artifacts.r-hub.io/", id),
+             stringsAsFactors = FALSE)
```

Comment:
I think the `paste0()` calls can be vectorized, and there  is no need for `rbind()`.

Reply:
:woman_facepalming: thanks, now fixed!